bin/check-puppet-last-run.rb

```diff
@@ -95,7 +107,11 @@ def run
       @message += " with #{@failures} failures"
     end
 
-    if @now - @last_run > config[:crit_age] || @failures > 0
+    if @restart_failures > 0
```

Comment:
This should probably be `if @restart_failures > 0 && config[report_restart_failures:]`